apps/hpc-cdm/src/i18n/langs/en.json

```diff
@@ -1,6 +1,6 @@
 {
   "switchToThisLanguage": "Switch to English",
-  "title": "Humanitarian Coordination Mapping - 2020",
+  "title": "Humanitarian Coordination Mapping - 2021",
```

Comment:
Why is the year hardcoded?

`<title>` element in `apps/hpc-cdm/src/index.html` has a reference to 2020 still.